components/esp_lvgl_port/docs/performance.md

```diff
@@ -0,0 +1,115 @@
+# LCD & LVGL Performance
+
+This document provides steps, how to set up your LCD and LVGL port for the best performance and comparing the difference settings. All settings and measurements are valid for Espressif's chips.
+
+## Process of the performance measuring
+
+There are two main processes for measurement performance on LCD:
+
+1. Measure time, how long it takes to refresh the whole screen.
+2. Use `lv_demo_benchmark()` to measure FPS.
```

Comment:
We can also add this: 3. Use `lv_demo_music()` (with autoplay) to measure FPS.  
Because the result of benchmark demo is weighted FPS (more like score), and the result of music demo is average FPS. Many users may care about actual FPS.

Reply:
And the configuration `LV_DISP_DEF_REFR_PERIOD` affects the results of performance testing very much. Though reducing the value of this param won't improve actual LVGL performance normally, it will increase weighted or average FPS. We should mention it.

Reply:
Thank you for info. I will add it into readme and add it into my example table too

Reply:
Updated documentation.